app/assets/javascripts/utilities/buildArticleHTML.js

```diff
@@ -49,18 +49,15 @@ function buildArticleHTML(article, currentUserId = null) {
   }
 
   if (article && article.class_name === 'Organization') {
-    return `<article class="crayons-story">
+    const html = `
+      <article class="crayons-story">
         <div class="crayons-story__body flex items-start gap-2">
           <a href="${article.slug}" class="crayons-podcast-episode__cover">
-            <img src="${article.profile_image.url}" alt="${
-      article.name
```

Comment:
I’m wondering, is there any way to keep the alt tag but still escape the name using a regex or something? Hoping not to erode accessibility in the name of security where possible.

Reply:
It's still here only moved to line 78